demo/app/styles.scss

```diff
@@ -50,3 +50,7 @@ body {
 .holiday {
   background-color: rgba(250, 246, 225, 0.5);
 }
+
+.lunch {
+  background-color: rgba(255, 0, 0, 0.5) !important; /* we need to overwrite the weekend rule */
```

Comment:
I think that with this feature implemented, we can remove the weekend styling, both from this repo codebase and from the style.scss file.